### DISCUSSION TITLE
[solved] Phonebox-plugin, TemplateSyntaxError: 'applied_filters' did not receive value(s) for the argument(s): 'query_params'

### DISCUSSION THREAD

**Discussion by Nikita-Fomichev-DiDi** [netbox-community/netbox#11568] [Q&A]
Hello everyone!

**Python version**: 3.9.5
**NetBox version**: 3.4.2
**psql --version**
psql (PostgreSQL) 13.5 (Ubuntu 13.5-0ubuntu0.21.04.1)
**redis-server --version**
Redis server v=6.0.11 sha=00000000:0 malloc=jemalloc-5.2.1 bits=64 build=83fe9b039c768864
**Phonebox-plugin** 0.0.4b1

![image](https://user-images.githubusercontent.com/123448748/214244570-3b153c1a-d068-4e31-8510-a02e63823356.png)

After install phonebox-plugin An exception appears when logging into the plugin.
The error, I believe, appeared after the transition of netbox from version 3.1.9 to 3.4.2, but I noticed it only now. The plugin was reinstalled according to the instructions in README.md 
I checked other discussions on the forum, I didn't find anything similar. Please clarify what I missed and where and how it can be fixed.

**Comment by PVM-NL** [comment]
More have the same problemn think it has to do with this:
https://github.com/iDebugAll/phonebox_plugin/issues/27

Support for 3.4.x isnt added yet by the maintainer.

**Reply by Nikita-Fomichev-DiDi** [reply]
this answer helped me:
https://github.com/iDebugAll/phonebox_plugin/issues/27#issuecomment-1362511544
